src/custom/utils/signatures.ts

```diff
@@ -1,15 +1,6 @@
-import {
-  domain as domainGp,
-  signOrder as signOrderGp,
-  signOrderCancellation as signOrderCancellationGp,
-  EcdsaSignature,
-  Order,
-  OrderCancellation as OrderCancellationGp,
-  Signature,
-  TypedDataVersionedSigner,
-  IntChainIdTypedDataV4Signer,
-  SigningScheme,
-} from '@cowprotocol/contracts'
+import { domain as domainGp } from '@cowprotocol/contracts'
+import type { Order, EcdsaSignature, Signature } from '@cowprotocol/contracts'
+import { EcdsaSigningScheme, SigningScheme, OrderParameters } from '@cowprotocol/cow-sdk'
 
 import { SupportedChainId as ChainId } from 'constants/chains'
```

Comment:
Shouldn't this file be exposed in the SDK as a SigningLib or something?

Reply:
We use to provide a simple way to sign using the SDK. 

https://docs.cow.fi/cow-sdk/sign-and-post-orders/step-2-sign-the-order

Reply:
https://github.com/cowprotocol/cowswap/pull/2180

Reply:
>We use to provide a simple way to sign using the SDK.

I would like to discuss it. There are pros and cons. '@cowprotocol/contracts' already contains everything necessary and it looks a little excessive to proxy the package from SDK. On the other hand, I agree, it's more clearly to export signing utils from SDK.
Let's discuss it in the next call.